packages/plugin-amplitude-node/lib/index.ts

```diff
@@ -14,6 +16,8 @@ export { AmplitudeOptions };
 export class AmplitudePlugin extends Plugin {
   private amplitude?: Amplitude;
 
+  private logger: Logger | undefined;
```

Comment:
Since this same pattern is being used in each plugin how about creating a `LoggingPluginBase` class in `sdk`?

```
class LoggingPluginBase {
  protected logger: Logger | undefined;
  load(options: PluginLoadOptions) {
    this.logger = options.logger;
  }

  protected logAction(id: string, action: string, state?: string, message?: string) => this.logger?.debug(`${this.id}: ${action}${state ? `(${state})` : ''} ${id}: ${message}`);

 ...
}
```

Reply:
I've added `PluginLogger` instead of the `LoggingPluginBase`.